testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -552,11 +554,51 @@ def test_ClusterCollection_get_centroids(self, cc):
         assert cc.get_centroids() == [1, 3, 5], \
                      "ClusterCollection centroids aren't as expected"
 
-    def test_Cluster_add_metadata(self, cluster):
+    def test_cluster_add_metadata(self, cluster):
         metadata = cluster.elements*10
         cluster.add_metadata('test', metadata)
         assert np.all(cluster.metadata['test'] == metadata), \
                      "Cluster metadata isn't as expected"
+        metadata = np.append(metadata, 9)
+        error_message = ("Size of metadata is not equal to the "
+                        "number of cluster elements")
+        with pytest.raises(TypeError, match=error_message):
+            cluster.add_metadata('test2', metadata)
+
+    def test_empty_cluster(self):
+        empty_cluster = encore.Cluster()
+        assert empty_cluster.size == 0
+        assert np.size(empty_cluster.elements) == 0
+        assert empty_cluster.centroid is None
+        assert bool(empty_cluster.metadata) is False
+
+    def test_centroid_not_in_elements(self):
+        error_message = "Centroid of cluster not found in the element list"
+        with pytest.raises(LookupError, match=error_message):
+            encore.Cluster([38, 39, 40, 41, 42, 43], 99)
+
+    def test_metadata_size_error(self):
+        error_message = ('Size of metadata having label "label" is '
+                        'not equal to the number of cluster elements')
```

Comment:
As above, should be indented by one more whitespace
```suggestion
                         'not equal to the number of cluster elements')
```